demo/src/tests/mod.rs

```diff
@@ -275,6 +275,18 @@ pub fn hello() -> Result<(), ()> {
 		"Crab emoji: {}",
 		String::from_utf8(crab).unwrap_or_default()
 	);
+
+	Ok(())
+}
+
+pub fn arithmetic() -> Result<(), ()> {
+	let mut rng = rand::thread_rng();
+	let x = rng.gen::<f64>();
+	let y: f64 = x.exp();
+	let z: f64 = y.log(E);
+
+	println!("x = {}, e^x = {}, ln(e^x) = {}", x, y, z);
+
 	Ok(())
```

Comment:
Maybe add a check here if x and z are approximately the same instead of always returning `Ok(())`

Reply:
Good idea